### PR TITLE
Migrate to null safety

### PR DESCRIPTION
**Pull request by bananagag**
Tried to migrate the package to a null-safety version, please have a look on it.

### CODE REVIEW

File: example/test/widget_test.dart [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-import 'package:image_cropper_example/main.dart';
+import '../lib/main.dart';
```
Comment (JeyPay) [inline example/test/widget_test.dart:10]: Why using the relative path here ?

File: lib/src/options.dart [hunk 2]
```diff
@@ -60,76 +59,77 @@ class CropAspectRatio {
...
-      this.cropGridRowCount,
-      this.cropGridColumnCount,
-      this.cropGridStrokeWidth,
-      this.showCropGrid,
-      this.lockAspectRatio,
-      this.hideBottomControls,
-      this.initAspectRatio});
+  final CropAspectRatioPreset? initAspectRatio;
+
+  const AndroidUiSettings({
```
Comment (JeyPay) [inline lib/src/options.dart:114]: Avoid unnecessary formatting

File: pubspec.yaml [hunk 3]
```diff
@@ -3,6 +3,10 @@ description: A Flutter plugin for Android and iOS supports cropping images
 version: 1.3.1
 homepage: https://github.com/hnvn/flutter_image_cropper
 
+environment:
+  sdk: '>=2.12.0-0 <3.0.0'
+  flutter: ^1.12.13
```
Comment (JeyPay) [inline pubspec.yaml:8]: Change Flutter version ?

Comment (bananagag) [reply pubspec.yaml:8]: Which Flutter version should I change to?

Comment (JeyPay) [reply pubspec.yaml:8]: Since null-safety was introduced by 1.24.0-12.2.pre, I guess this should be the minimum Flutter requirement but not sure. Do Do you know if we can run an older Flutter SDK with a more recent Dart version ?

**Review by JeyPay — APPROVED** [review]
Tested on Flutter Channel dev, 2.1.0-10.0.pre, on Linux, and is working perfectly. Can be merged

**Comment by hnvn** [comment]
LGTM. Thanks all for your contribution.
